### ISSUE TITLE
cell value not showing after end cell edit with custom datalist template

### ISSUE THREAD

**Issue by CodeLiftSleep** [angular-ui/ui-grid#6755]
Cannot get value to display in cell after editing with this custom template.
If I leave out the ui-grid-editor directive in the input field, the value shows in the cell but it doesn't fire the end edit event, so the cell maintains focus no matter where I go. Is there something else I should be doing to ensure the value is displayed once the end cell event fires?  The event fires with ui-grid-editor directive in the input, as I paused inside the afterCellEdit event but the cell shows blank once it loses focus, although the value is actually there(when I click back in the cell to get focus, the value pops up).

Basically the purpose of this template is to allow users to enter an option if their option is not present in the dropdown(aka the datalist, a valid HTML5 type).

```
<div>
    <form name="inputForm">
        <input type="text" list="Con_list" ui-grid-editor ng-model="MODEL_COL_FIELD" />
        <datalist id="Con_list">
            <option ng-repeat="option in col.colDef.editDropdownOptionsArray"  value=" 
            {{option.Position}}">{{option.Position}}</option>
        </datalist>
    </form>
</div>
```

**Comment by mportuga** [comment]
Can you provide a plunker demonstrating the issue?

**Comment by WorkMAE** [comment]
Here is a plunk:

https://plnkr.co/edit/jlce89jjjG7w8qVrEp8V

The first grid is without the ui-grid-editor directive and does not lose focus after you click away.  It works and the value shows, it just doesn't look right.  The second grid has the ui-grid-editor directive in it but it is not having the same behavior as it does on the one I'm using at work...at work it loses focus properly, but then it doesn't display anything at all, although the value is there because if you click on it again, the value will be there.

**Comment by WorkMAE** [comment]
Why would it need any activity?  I have already provided what you asked for, how about just fix the problem?

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
